src/components/EnvironmentDetailsPageTabs.tsx

```diff
@@ -21,12 +19,10 @@ export const EnvironmentDetailsPageTabs: React.FC<EnvironmentDetailsPageTabsProp
 }) => {
   const { t } = useTranslation();
   const { appName } = match.params;
-  const [secretNS, secretName] = useDefaultSecret();
-  const pipelinesBaseURI = getPipelinesBaseURI(secretNS, secretName);
   const searchParams = new URLSearchParams(location.search);
   const manifestURL = searchParams.get('url');
-  const applicationBaseURI = getApplicationsBaseURI(appName, secretNS, secretName, manifestURL);
-  const [envs, emptyStateMsg] = useEnvDetails(appName, manifestURL, pipelinesBaseURI);
+  const applicationBaseURI = `/application/${appName}?url=${manifestURL}&app=${appName}`;
```

Comment:
After we pass this to the history page, we check if this is available.  I think we could remove that check there at line 52.  Also there, it looks like we are calling history API every 1-2 seconds. Could you check there?

Reply:
Thanks for the review. I missed the history API call, will let you know once I fix it. Good catch 👍